_posts/2020-08-19-cockpit-226-cockpit-podman-22.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 226 and Cockpit Podman 22
+author: mvo
+date: 2020-08-19
+tags: cockpit linux
+slug: cockpit-226
+category: release
+summary: Cockpit and Cockpit Podman news
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 226 and Cockpit Podman version 22.
+
+### Storage: Better support for "noauto" LUKS devices
+
+Cockpit now behaves more sensible with encrypted filesystems that have
+"noauto" in their LUKS options. Such a filesystem will now also always
+have "noauto" in their mounting options even when mounted.
```

Comment:
```suggestion
Cockpit  Encrypted filesystems with "noauto" in LUKS options now .... ??!??!?
```

This needs rewriting, but it doesn't say what Cockpit did before or what it does now.

Reply:
```suggestion
Cockpit now better respects the "noauto" option for encrypted filesystems.

Previously, Cockpit would force a "noauto" encrypted filesystem to mount at next boot when it was simply mounted within Cockpit. Now, a filesystem with the "noauto" settings will remain unmounted.
```

Reply:
Another try:

```suggestion
Cockpit now better respects the LUKS "noauto" option for encrypted filesystems.

Previously, Cockpit would force a encrypted filesystem to mount at next boot when it was simply mounted within Cockpit. Now, encrypted filesystems set with "noauto" will remain unmounted during boot.

Please [refer to issue 14467](https://github.com/cockpit-project/cockpit/issues/14467) for more details.
```